lib/Doctrine/ODM/PHPCR/Query/Builder/ConverterPhpcr.php

```diff
@@ -145,13 +145,17 @@ protected function getPhpcrProperty($originalAlias, $odmField)
 
         $propertyName = $fieldMeta['property'];
 
+        if ($fieldMeta['type'] == 'uuid') {
```

Comment:
https://github.com/doctrine/phpcr-odm/blob/master/lib/Doctrine/ODM/PHPCR/Mapping/ClassMetadata.php#L1407 should lead to this condition never being true